lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -380,6 +381,24 @@ class ReportUIFeatures {
     this._document.body.removeChild(a);
     setTimeout(_ => URL.revokeObjectURL(href), 500);
   }
+
+  /**
+   * Register listeners to toggle dark theme via clicking on LH logo or pressing 'd'.
+   * @private
+   */
+  _registerDarkThemeToggles() {
+    // TODO(#8185) - Persist dark mode state.
```

Comment:
I don't see how this would be easily possible across clients but I guess we'll see :)

```suggestion
    // TODO(Hoten) - Persist dark mode state - https://github.com/GoogleChrome/lighthouse/issues/8185
```